projects/fsastorefrontlib/src/cms-components/form/general-information/general-information.component.ts

```diff
@@ -59,12 +63,15 @@ export class GeneralInformationComponent extends FormCMSComponent
         .pipe(
           map(cart => {
             if (cart.code) {
-              const chooseCoverFormId = <any>(
-                (<FSCart>cart).insuranceQuote?.quoteDetails?.formId
-              );
-              if (chooseCoverFormId) {
-                this.formDataService.loadFormData(chooseCoverFormId);
-                this.formData$ = this.formDataService.getFormData();
+              const bindingState = (<FSCart>cart).insuranceQuote.state.code;
+              if (bindingState !== BindingStateType.BIND) {
+                const chooseCoverFormId = <any>(
+                  (<FSCart>cart).insuranceQuote?.quoteDetails?.formId
+                );
+                if (chooseCoverFormId) {
```

Comment:
Could we move this in the same, first if block?
`bindingState !== BindingStateType.BIND && chooseCoverFormId`